backend/molgenis-emx2-sql/src/main/resources/org/molgenis/emx2/sql/migration19.sql

```diff
@@ -0,0 +1,16 @@
+DO $$
+    DECLARE
+        schema_name text;
+    BEGIN
+        -- Loop through each schema in the current database
+        FOR schema_name IN
+            SELECT s.table_schema
+            FROM "MOLGENIS".schema_metadata s
+            LOOP
+                -- Execute the CREATE COLLATION statement for each schema
+                EXECUTE format(
+                        'CREATE COLLATION IF NOT EXISTS %I.numeric (provider = icu, locale = ''en-u-kn-true'');',
+                        schema_name
```

Comment:
only in 'MOLGENIS' and then refer to it with the namespace?

Reply:
yes good, idea , pr as is work fine on my machine , but not on the preview , i now suspect that maybe while trying stuff out i (accidentally) added the collation to the db level ( instead of schema level), i need to look at the docs more to see about the options, the multi level stuff and 'default' paths in pg always confuse me, adding it to a single sys schema ( molgenis ) also seems like a good idea